resources/kcp/charts/metris/values.yaml

```diff
@@ -42,6 +40,7 @@ provider:
   maxretries: 20
 
 edp:
+  namespace: "TBD"
```

Comment:
Is EDP namespace unknown yet?

Reply:
Yes, this varies with dev/stage/prod envs.